packages/safe-apps-sdk/src/safe/safe.test.ts

```diff
@@ -373,35 +373,5 @@ describe('Safe Apps SDK safe methods', () => {
 
       expect(postMessageSpy).toHaveBeenCalledWith(expect.objectContaining({ method: Methods.getChainInfo }), '*');
     });
-
-    test('should resolve the correct ChainInfo types', async () => {
```

Comment:
I removed this test because it was obsolete. We mock the return value and then compare if the return value equals the mocked value, it only tests if jest's mocking works